promql/promqltest/testdata/aggregators.test

```diff
@@ -229,13 +229,28 @@ load 5m
   http_requests{job="api-server", instance="0", group="canary"}		NaN
   http_requests{job="api-server", instance="1", group="canary"}		3
   http_requests{job="api-server", instance="2", group="canary"}		4
+  http_requests_histogram{job="api-server", instance="3", group="canary"} {{schema:2 count:4 sum:10 buckets:[1 0 0 0 1 0 0 1 1]}}
 
 eval instant at 0m max(http_requests)
   {} 4
 
+# should produce same results ignoring histograms with info annotations.
```

Comment:
Start sentence with a capital letter .

In different news, I would change the word order to avoid misunderstandings. Currently, it sounds like histograms are only ignored if they have an info annotation (of course, histograms don't have annotations, but it's still hard to read).

Maybe like this:

```suggestion
# The histogram is ignored here so the result doesn't change but it has an info annotation now.
```

Let's then word the other test comments accordingly.